tests/mock_network/utils.rs

```diff
@@ -322,6 +325,41 @@ pub fn consensus_reached(
     *current_count >= expected_count
 }
 
+// Returns whether all elders of section `a` know at least `threshold` online nodes from
+// section `b`.
+pub fn section_knowledge_is_up_to_date(
+    nodes: &[TestNode],
+    a: &Prefix<XorName>,
+    b: &Prefix<XorName>,
+    threshold: usize,
+) -> bool {
+    let names_b: Vec<_> = nodes_with_prefix(nodes, b)
```

Comment:
just wondering that for `a`, it is the function of `elders_with_prefix` being used, but here is `nodes_with_prefix`, is it ok?

Reply:
Yes, that's on purpose. It's enough to know nodes of any type, not necessarily elders. If we only know non-elders, we can still send them a message and the non-elders will relay it to their elders so everything still works.

Reply:
ah, ok then